mockOSApi.v2/Models/Security/Group.cs

```diff
@@ -0,0 +1,23 @@
+namespace mockOSApi.Models;
+
+public class Group
+{
+
+    public int Gid { get; set; }
+    public string Name { get; set; }
+
+    public Capabilities Capabilities { get; set; }
```

Comment:
I think this should be Capabilities[] instead, and the enum should be named `Capability`, or `Policy` or something of the sort

Reply:
done